### ISSUE TITLE
Default shortcuts/key bindings

### ISSUE THREAD

**Issue by MelvIsntNormal** [Bismuth-Forge/bismuth#371]
Apologies if this is in the repo somewhere, I tried looking but couldn't find anything that seemed to answer this.

I've just installed this plugin on Arch Linux through the AUR. When I go into `System Settings > Shortcuts > Window Tiling`, I see all the possible shortcuts there, but no key bindings (each entry has "No active shortcuts" next to them). Are there supposed to be any default key bindings, or am I expected to make my own? I was hoping for some kind of default I could tweak if needed; I'm having a bit of decision paralysis trying to build my own set of keybindings from scratch...

**Comment by gikari** [comment]
There are indeed some default shortcuts. 

Could you reproduce the problem, when you install Bismuth from source?

**Comment by MelvIsntNormal** [comment]
Yep, I managed to reproduce this with the source. I uninstalled the AUR package, followed the CONTRIBUTING.md, and got the same result

**Comment by MelvIsntNormal** [comment]
[kglobalshortcutsrc.txt](https://github.com/Bismuth-Forge/bismuth/files/8891764/kglobalshortcutsrc.txt).

**Comment by gikari** [comment]
Try to delete the `[bismuth]` section in that config, then restart kglobalaccel daemon and Bismuth.

If this didn't help, check if Bismuth shortcuts are not overridden by something else. You can see the default shortcuts in `[bismuth]` section. I see "I3-Tiling" and `Tiling` shortcuts in your config, so that they could do some.

In that case, you should remove that lines in the config, then restart the kglobalaccel and disable the kwin script, that added them.

**Comment by MelvIsntNormal** [comment]
I'm still having the same issue unfortunately.

I (think I) removed every `i3` and `TILING` entry in the `kglobalshortcutsrc`, and removed the bismuth section, but I still cannot see the shortcuts show up in system settings.

However, I do now see the shortcuts in the `kglobalshortcutsrc`. But they're not being recognised by my system.

[kglobalshortcutsrc.1.txt](https://github.com/Bismuth-Forge/bismuth/files/8892262/kglobalshortcutsrc.1.txt)

**Comment by kuba160** [comment]
I experienced same issue after installing (on Arch), but only on Wayland session. Shortcuts are there, but they are all disabled by default. On X11 session default shortcuts get applied and they stay on when switching back to Wayland session again.

**Comment by Tom-Welch** [comment]
I'm no expert so please excuse any ignorance upfront.

I just ran into the same issue, also running on Arch Linux & Wayland. It looks as if the default shortcuts have 'none' before the assigned shortcut keys in `kglobalshortcutsrc` so they appear to be ignored / disabled.

Example; 
`decrease_master_win_count=none,Meta+[`

My workaround was to find and replace `none,` under the block of bismuth shortcuts, after logging out and back in, all the shortcuts appeared under the settings menu and started working.

**Comment by nmstoker** [comment]
@Tom-Welch - thanks a lot for your comment!  I expect your workaround would have worked for me but what it appears to be doing is shifting the setting for the _default shortcut_ into the position for the actual shortcut.

So that I could keep the default shortcut entry as an option in the UI to fall back on, I copied all the values that weren't blank directly after the none, and pasted them in place of the none on each such row; I then logged out and back in and it worked as desired.

In case it saves others this cut n paste exercise here are the entries I had after doing this:

```` ini
[bismuth]
_k_friendly_name=Window Tiling
decrease_master_size=none,,Decrease Master Area Size
decrease_master_win_count=Meta+[,Meta+[,Decrease Master Area Window Count
decrease_window_height=Meta+Ctrl+K,Meta+Ctrl+K,Decrease Window Height
decrease_window_width=Meta+Ctrl+H,Meta+Ctrl+H,Decrease Window Width
focus_bottom_window=Meta+J,Meta+J,Focus Bottom Window
focus_left_window=Meta+H,Meta+H,Focus Left Window
focus_next_window=none,,Focus Next Window
focus_prev_window=none,,Focus Previous Window
focus_right_window=Meta+L,Meta+L,Focus Right Window
focus_upper_window=Meta+K,Meta+K,Focus Upper Window
increase_master_size=none,,Increase Master Area Size
increase_master_win_count=Meta+],Meta+],Increase Master Area Window Count
increase_window_height=Meta+Ctrl+J,Meta+Ctrl+J,Increase Window Height
increase_window_width=Meta+Ctrl+L,Meta+Ctrl+L,Increase Window Width
move_window_to_bottom_pos=Meta+Shift+J,Meta+Shift+J,Move Window Down
move_window_to_left_pos=Meta+Shift+H,Meta+Shift+H,Move Window Left
move_window_to_next_pos=none,,Move Window to the Next Position
move_window_to_prev_pos=none,,Move Window to the Previous Position
move_window_to_right_pos=Meta+Shift+L,Meta+Shift+L,Move Window Right
move_window_to_upper_pos=Meta+Shift+K,Meta+Shift+K,Move Window Up
next_layout=Meta+\\\\,Meta+\\\\,Switch to the Next Layout
prev_layout=Meta+|,Meta+|,Switch to the Previous Layout
push_window_to_master=Meta+Return,Meta+Return,Push Active Window to Master Area
rotate=Meta+R,Meta+R,Rotate
rotate_part=Meta+Shift+R,Meta+Shift+R,Rotate Part
rotate_reverse=none,,Rotate (Reverse)
toggle_float_layout=Meta+Shift+F,Meta+Shift+F,Toggle Floating Layout
toggle_monocle_layout=Meta+M,Meta+M,Toggle Monocle Layout
toggle_quarter_layout=none,,Toggle Quarter Layout
toggle_spiral_layout=none,,Toggle Spiral Layout
toggle_spread_layout=none,,Toggle Spread Layout
toggle_stair_layout=none,,Toggle Stair Layout
toggle_three_column_layout=none,,Toggle Three Column Layout
toggle_tile_layout=Meta+T,Meta+T,Toggle Tile Layout
toggle_window_floating=Meta+F,Meta+F,Toggle Active Window Floating
````

FYI, my installation was via AUR just a few days ago. I haven't had time to try to replicate by installing directly from source.
